ray_on_golem/server/cluster/nodes.py

```diff
@@ -435,13 +439,47 @@ async def _on_monitor_check_failed(
 class HeadClusterNode(WorkerClusterNode):
     """Self-contained element that represents explicitly a Ray head node."""
 
-    webserver_port: int
+    _webserver_port: int
```

Comment:
why not leave these attributes as regular ones, without making them "private"?

Reply:
Previously that port was used externally, but not anymore. Now it exists only for the sake of sidecar, also making it private does not generate additional data in communication via http.